modules/social_features/social_core/social_core.module

```diff
@@ -977,6 +980,31 @@ function social_core_form_node_form_alter(&$form, FormStateInterface $form_state
   }
 }
 
+/**
+ * Custom form validation handler for validating created date.
+ *
+ * @param array $form
+ *   The form array.
+ * @param \Drupal\Core\Form\FormStateInterface $form_state
+ *   Form state.
+ *
+ * @throws \Drupal\Core\Entity\EntityStorageException
+ */
+function _social_core_created_date_validation(array $form, FormStateInterface $form_state) {
```

Comment:
@nkoporec Is it not possible to add a return type to this function to not have the additional message in the baseline file?

```suggestion
function _social_core_created_date_validation(array $form, FormStateInterface $form_state): void {
```